src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,375 @@
+import React from 'react'
+import {Box} from '..'
+import {useResponsiveValue, ResponsiveValue} from '../hooks/useResponsiveValue'
+import {SxProp, merge, BetterSystemStyleObject} from '../sx'
+import Heading from '../Heading'
+import {ArrowLeftIcon} from '@primer/octicons-react'
+import Link from '../Link'
+import {ForwardRefComponent as PolymorphicForwardRefComponent} from '../utils/polymorphic'
+const REGION_ORDER = {
+  ContextArea: 0,
+  TitleArea: 1,
+  Description: 2,
+  Navigation: 3,
+}
+
+// Types that are shared between sub components
+export type sharedPropTypes = {
+  hidden?: boolean | ResponsiveValue<boolean>
+} & SxProp
+
+// Default state for the `visible` prop when a sub component is only visible on narrow viewport
+const hiddenOnRegularAndWide = {
+  narrow: false,
+  regular: true,
+  wide: true,
+}
+
+// Default state for the `visible` prop when a sub component is visible on regular and wide viewport
+const hiddenOnNarrow = {
+  narrow: true,
+  regular: false,
+  wide: false,
+}
+
+// Root
+// -----------------------------------------------------------------------------
+export type PageHeaderProps = {
+  'aria-label'?: React.AriaAttributes['aria-label']
+  as?: React.ElementType | 'header' | 'div'
+} & sharedPropTypes
+
+const Root: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({children, sx = {}, as = 'div'}) => {
+  const rootStyles = {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '0.5rem',
+  }
+  return (
+    <Box as={as} sx={merge<BetterSystemStyleObject>(rootStyles, sx)}>
+      {children}
+    </Box>
+  )
+}
+
+// PageHeader.ContextArea : Only visible on narrow viewports by default to provide user context of where they are at their journey. `visible` prop available
+// to manage their custom visibility but consumers should be careful if they choose to hide this on narrow viewports.
+// PageHeader.ContextArea Sub Components: PageHeader.ParentLink, PageHeader.ContextBar, PageHeader.ContextAreaActions
+// ---------------------------------------------------------------------
+
+const ContextArea: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({
+  children,
+  hidden = hiddenOnRegularAndWide,
+  sx = {},
+}) => {
+  const isHidden = useResponsiveValue(hidden, false)
+  const contentNavStyles = {
+    display: isHidden ? 'none' : 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: '0.5rem',
+    order: REGION_ORDER.ContextArea,
+  }
+  return <Box sx={merge<BetterSystemStyleObject>(contentNavStyles, sx)}>{children}</Box>
+}
+// adopted from React.AnchorHTMLAttributes
+type LinkProps = {
```

Comment:
Is there a reason we can't do this instead?
```
type LinkProps = Pick<
  React.AnchorHTMLAttributes,  'download' |
  'href' |
  'hrefLang' |
  'media' |
  'ping' |
  'rel' |
  'target' |
  'type' |
  'referrerPolicy'
>
```

(non-blocking feedback)

Reply:
I don't think so. It seemed a bit of a convention to me in the code base so I copied over. I updated now like below

```
type LinkProps = Pick<
  React.AnchorHTMLAttributes<HTMLAnchorElement>,
  'download' | 'href' | 'hrefLang' | 'media' | 'ping' | 'rel' | 'target' | 'type' | 'referrerPolicy'
>
```

Reply:
I think either way works. I wasn't aware the original version was an established convention in the codebase